package.json

```diff
@@ -42,9 +42,10 @@
     "@types/tmp": "0.1.0",
     "@types/url-template": "^2.0.28",
     "@types/uuid": "^7.0.0",
+    "c8": "^7.0.0",
     "chai": "^4.2.0",
     "codecov": "^3.5.0",
-    "gts": "^1.0.0",
+    "gts": "^2.0.0-alpha.5",
```

Comment:
mind bumping this up to `alpha.9` (npm i gts@latest --save-dev) and running `npm run fix`.